test/integration/services/data/test-update-reduction.js

```diff
@@ -49,6 +52,16 @@ describe('/services/data/update-reduction', function () {
       })
   })
 
+  it('should update a reduction status and return an id ', function () {
+    return updateReductionStatus(addedReductionId, workloadOwnerId, reductionStatusType.ARCHIVED)
+      .then(function (result) {
+        // Store the id so that we can delete it after the test is complete
+        reductionResult.id = result
+        expect(result[0]).to.be.a('number')
+        expect(result).to.eql(addedReductionId)
+      })
+  })
+
```

Comment:
So, we now have a method to get reductions. It would be good if we could have a test that updated the reduction status, returned the id, then we get the reduction based on that id and verified that the status is correct.

Reply:
Agreed, was thinking about that :-)